apps/projects/app/components/Panel/NewProject/NewProject.js

```diff
@@ -113,110 +121,187 @@ const NewProject = () => {
+              <TextInput
+                type="search"
+                placeholder="Search for a repository"
+                wide
+                value={filter}
+                onChange={updateFilter}
+                adornment={
+                  filter === '' && (
+                    <IconSearch
+                      css={`
+                        color: ${theme.surfaceOpened};
+                        margin-right: ${GU}px;
+                      `}
                     />
+                  )
+                }
+                adornmentPosition="end"
+                ref={searchRef}
+                aria-label="Search"
+              />
+
+              <ScrollableList>
+                <RepoList visibleRepos={visibleRepos} repoArray={repoArray} />
+              </ScrollableList>
+
+            </div>
+          )
+        }
+
+        if (loading) return (
+          <RepoInfo>
+            <LoadingAnimation />
+            <div>Loading repositories...</div>
+          </RepoInfo>
+        )
+
+        if (error) return (
+          <RepoInfo>
+            <Text size="xsmall" style={{ margin: '20px 0' }}>
+              Error {JSON.stringify(error)}
+            </Text>
+            <Button wide mode="strong" onClick={() => refetch()}>
+              Try refetching?
+            </Button>
+          </RepoInfo>
+        )
+      }}
+    </Query>
+  )
 
-                    <ScrollableList>
-                      <RepoList visibleRepos={visibleRepos} repoArray={repoArray} />
-                    </ScrollableList>
-
-                    <Info css={`margin: ${3 * GU}px 0`}>
-                        Projects in Aragon are a one-to-one mapping to a GitHub repository.
-                        You’ll be able to prioritize your backlog, reach consensus on issue
-                        valuations, and allocate bounties to multiple issues.
-                    </Info>
-
-                    <Button
-                      mode="strong"
-                      wide
-                      onClick={handleNewProject}
-                      disabled={repoSelected < 0}
-                    >
-                      Submit
-                    </Button>
-                  </div>
-                )
-              }
-
-              if (loading) return (
-                <RepoInfo>
-                  <LoadingAnimation />
-                  <div>Loading repositories...</div>
-                </RepoInfo>
-              )
-
-              if (error) return (
-                <RepoInfo>
-                  <Text size="xsmall" style={{ margin: '20px 0' }}>
-                      Error {JSON.stringify(error)}
-                  </Text>
-                  <Button wide mode="strong" onClick={() => refetch()}>
-                      Try refetching?
-                  </Button>
-                </RepoInfo>
-              )
-            }}
-          </Query>
+  return (
+    <>
+      {github.status !== 'authenticated' ? (
```

Comment:
As it is now, this isn't accessible since the _initial_ splash screen requires auth. We need to go straight into project view.